Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:latest
```

Comment:
You can build on host machine and run on `alpine`. Then use something like this to copy binary and related json config files into `dist` folder. Then COPY `dist` into Docker container.
```
build: ## build the app and moves it to dist/ folder
	GOOS=linux CGO_ENABLED=0 go build -s -o task_app
	mkdir -p ./dist
	mv task_app ./dist
```

This update is not important, please do this at the very end